lib/pages/faq/QuestionsList.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:acs_upb_mobile/pages/faq/model/question.dart';
+import 'package:acs_upb_mobile/widgets/selectable.dart';
+import 'package:dynamic_text_highlighting/dynamic_text_highlighting.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:markdown/markdown.dart' as md;
+
+class QuestionsList extends StatefulWidget {
```

Comment:
Move this widget to `faq_page.dart`.  In dart, widgets don't each need to have their own separate file - I generally prefer to have one file per in-app page.